### DISCUSSION TITLE
Modification made on 2 sensors

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [djansen1987/SAJeSolar#40] [General]
After integrating this code into HA, I have noticed that the graphs of esolar pvPower and esolar batCurr are not displayed. Looking at the code I saw that they do not have the unit of measure assigned. I modified these descriptions by adding the lines as shown here, and now I see the corresponding graphics.
 
SensorEntityDescription(
         key="pvPower",
         name="pvPower",
         icon="mdi:solar-panel-large",
         native_unit_of_measurement=POWER_WATT,
     ),

SensorEntityDescription(
         key="batCurr",
         name="batCurr",
         icon="mdi:solar-panel-large",
         native_unit_of_measurement="A",
     ),

**Comment by djansen1987** [comment]
pvPower was already added, also updated batCurr
update in release:
https://github.com/djansen1987/SAJeSolar/releases/tag/v1.4.6

Please mark as answered if this helps
